measure_extinction/plotting/plot_ext.py

```diff
@@ -202,7 +202,7 @@ def plot_extmodels(extdata, alax=False):
             # compute A(V)
             extdata.calc_AV()
             # convert the model curve from A(lambda)/A(V) to E(lambda-V), using the computed A(V) of the data.
```

Comment:
We should probably check first if A(V) is already in extdata.columns, instead of just calculating it.

Reply:
Good point.  Added a check.